src/components/Image/AsPosts/GalleryModerationModal.tsx

```diff
@@ -26,16 +27,40 @@ import { Flags } from '~/shared/utils';
 import { useDialogContext } from '~/components/Dialog/DialogProvider';
 import { createDebouncer } from '~/utils/debouncer';
 import { TagSort } from '~/server/common/enums';
+import { openConfirmModal } from '@mantine/modals';
 
 export function GalleryModerationModal({ modelId }: { modelId: number }) {
   const dialog = useDialogContext();
+  const { copySettings, updating, copyLoading } = useGallerySettings({ modelId: modelId });
+
+  const handleCopySettings = () => {
+    openConfirmModal({
+      title: 'Copy Gallery Moderation Preferences',
+      centered: true,
+      children:
+        'This will copy the gallery moderation preferences from this model to all your models and future ones. Are you sure you want to proceed?',
+      onConfirm: async () => {
+        await copySettings(modelId);
+      },
+      labels: { confirm: 'Yes, continue', cancel: 'No, cancel' },
+    });
+  };
 
   return (
     <Modal {...dialog} title="Gallery Moderation Preferences">
       <Stack>
         <HiddenTagsSection modelId={modelId} />
         <HiddenUsersSection modelId={modelId} />
         <MatureContentSection modelId={modelId} />
+
+        <Button
+          variant="outline"
+          size="xs"
+          onClick={handleCopySettings}
+          loading={updating || copyLoading}
+        >
+          Copy settings
```

Comment:
`Apply this settings to all my models` sounds like a better fit for this button. 
Also, we could have a `Make this my default Settings` so that they are applied only to **future** models.

Having `Copy settings` made me think this'd be from one model to another 😬